verifier/src/main/kotlin/tech/figure/classification/asset/verifier/config/EventStreamProvider.kt

```diff
@@ -1,16 +1,14 @@
 package tech.figure.classification.asset.verifier.config
 
-import io.provenance.eventstream.stream.clients.BlockData
 import tech.figure.classification.asset.verifier.provenance.AssetClassificationEvent
 
 interface EventStreamProvider {
     suspend fun currentHeight(): Long?
     suspend fun startProcessingFromHeight(
```

Comment:
I was actually thinking this function itself could respond with a recoverable status.  That way the "restarter" could determine whether or not a restart was possible and whatnot after the function terminates naturally, as opposed to emitting it in the `onError` block, which could independently occur elsewhere

Reply:
updated, but let me know if thats not what you meant D: